### PR TITLE
Add timer and shape movement to UI

### PR DESCRIPTION
**Pull request by JRojas999**

### CODE REVIEW

File: app/src/main/java/src/utilities/Interface.java [hunk 1]
```diff
@@ -84,20 +90,62 @@ public void keyTyped(KeyEvent e) {
...
+            public void run() {
+                updateShape(Color.black);
+                shape.moveDown();
+                System.out.println("pos y: " + shape.getyPosition());
+                updateShape(Color.green);
+            }
+        };
+        timer.scheduleAtFixedRate(task, 0, SPEED);
+    }
+    /** */
```
Comment (Raymundogs) [inline app/src/main/java/src/utilities/Interface.java]: Missing method description

File: app/src/main/java/src/utilities/Interface.java [hunk 2]
```diff
@@ -84,20 +90,62 @@ public void keyTyped(KeyEvent e) {
      */
     @Override
     public void keyPressed(final KeyEvent e) {
-//        if (e.getKeyCode() == KeyEvent.VK_RIGHT) {
```
Comment (JessickaMoya) [inline app/src/main/java/src/utilities/Interface.java:87]: There is commented code

File: app/src/main/java/src/utilities/Interface.java [hunk 3]
```diff
@@ -84,20 +90,62 @@ public void keyTyped(KeyEvent e) {
...
+        for (int i = 0; i < shape.getContainer().length - shape.getDownRows(); i++) {
+            for (int j = shape.getLeftColumns(); j < shape.getContainer()[0].length - shape.getRightColumns(); j++) {
+                if (shape.getContainer()[i][j]) {
+                    updateShape3(this.labelArray[i + y][j + x], color);
+                }
+            }
+        }
+    }
+
+    /** */
```
Comment (Raymundogs) [inline app/src/main/java/src/utilities/Interface.java]: Missing method description

**Review by jpgonzales27 — APPROVED** [review]

**Review by JessickaMoya — APPROVED** [review]

File: app/src/main/java/src/utilities/Interface.java [hunk 4]
```diff
@@ -84,20 +90,62 @@ public void keyTyped(KeyEvent e) {
...
+    public void updateShape3(final JLabel label, final Color color) {
```
Comment (Raymundogs) [inline app/src/main/java/src/utilities/Interface.java]: Why updateShape3?

**Review by Raymundogs — APPROVED** [review]

File: app/src/main/java/src/utilities/Interface.java [hunk 5]
```diff
@@ -84,20 +90,62 @@ public void keyTyped(KeyEvent e) {
...
+    /**
+     * Method to initialize timer.
+     */
+    public void start() {
+        TimerTask task = new TimerTask() {
+            @Override
+            public void run() {
+                updateShape(Color.black);
+                shape.moveDown();
+                System.out.println("pos y: " + shape.getyPosition());
```
Comment (jpgonzales27) [inline app/src/main/java/src/utilities/Interface.java]: Maybe this syso should be removed

**Review by JessickaMoya — APPROVED** [review]

**Pull request merged**